src/components/MaintenanceInfo/MaintenanceInfoLayout.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Layout} from '../../pages/Layout/Layout';
+import styles from './MaintenanceInfoLayout.module.scss';
+import image from '../../assets/images/broken_server.png';
+import {useMount} from '../../hooks/useMount';
+import {useState} from 'react';
+import {getMaintenances} from '../../api';
+import {Maintenance} from '../../types/Maintenance';
+
+const ServiceMessage = (props: Maintenance) => {
+  if (props.isMaintenanceMode) {
+    if (props.isRestorationTimeKnown) {
+      return <div>Our servers are currently down for maintenance and are scheduled to be back up
+        at {props.restorationTime}</div>;
+    } else {
+      return <div>Our servers are currently down for maintenance. Please check back later.</div>;
+    }
+  }
+  else {
+    return <div>serviceMessage</div>;
```

Comment:
This looks like a typo, it will literally say "serviceMessage" which won't make sense to the end user.

Reply:
oh whoops, I think this was a debugging thing I forgot to change back